support/support.js

```diff
@@ -0,0 +1,183 @@
+(function() {
+    'use strict';
+
+    var support = angular.module('support', [
+        'ngMaterial',
+        'ui.router',
+        'ngAnimate',
+        'firebase',
```

Comment:
@ruanres I think that variable comes from here..

Reply:
:+1: